docs/source/api_reference.rst

```diff
@@ -32,14 +32,6 @@ Wrappers
 ~~~~~~~~
 .. currentmodule:: featuretools
 
-Scikit-learn (BETA)
```

Comment:
We can also remove the four lines above this to get rid of the `Wrappers` section

Reply:
Fixed